Dockerfile

```
@@ -59,28 +59,33 @@
-# - libgcc for runtime C dependencies
-RUN apk add --no-cache \
-      ncurses-libs=6.4_p20240420-r0 \
-      libstdc++=13.2.1_git20240309-r0 \
-      openssl=3.3.2-r0 \
-      ca-certificates=20240705-r0 \
-      libgcc=13.2.1_git20240309-r0 \
- && addgroup -S app \
- && adduser -S -G app app
+# - libgcc-s1 for runtime C dependencies
+# - wget for HEALTHCHECK
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)